src/system/system.cosma.make

```diff
@@ -0,0 +1,42 @@
+# system.make for cosma8 (apr 2023)
+# For user docs see https://cosma.readthedocs.io/en/latest/
+# load these modules
+# module load intel_comp/2022.3.0 compiler mpi mkl
```

Comment:
We should probably remove comments about modules to load, unless those comments are kept up-to-date...